app/assets/javascripts/discourse/app/components/sidebar/footer.gjs

```diff
@@ -35,6 +36,16 @@ export default class SidebarFooter extends Component {
     this.modal.show(SidebarSectionForm);
   }
 
+  @action
+  showKeyboardShorcuts() {
```

Comment:
🩳 

```suggestion
  showKeyboardShortcuts() {
```

Reply:
weird I tried it, wondered how it works 🤔

Reply:
A pair of typos make it right! 😄

Reply:
ah yes ofc, for some reason I thought it was only one